Revisions to hab license (#1917)
chef/chef-web-docs: chef_master/source/chef_license_accept.rst

Habitat
-----------------------------------------------------

Two methods are generally used to accept the Chef Habitat license:

> #. Users can pass the ``hab license accept`` or ``hab license accept-no-persist`` argument in the command line.

Reviewer: `hab license accept-no-persist` is not an available command.

```suggestion
#. Users can execute ``hab license accept`` on the command line.
```
